OrbitGl/TimeGraph.cpp

```diff
@@ -211,6 +211,25 @@ void TimeGraph::PanTime(int a_InitialX, int a_CurrentX, int a_Width,
   NeedsUpdate();
 }
 
+void TimeGraph::VerticallyScrollIntoView(const TextBox* text_box) {
+  CHECK(text_box != nullptr);
+  if (text_box->GetPos() == Vec2::Zero())  // Have to update the information of
```

Comment:
The logic here seems incorrect. NeedsUpdate() does not do any update, it only sets a flag that will trigger an update later. So, NeedsUpdate() here doesn't help in this function call. 

Further indication that there is something wrong here: You call NeedsUpdate unconditionally below, so the overall effect of NeedsUpdate calls by the current function is unchanged independent of whether` text_box->GetPos() is Vec2::Zero()`.